### PR TITLE
CONTRIBUTING.md Quick Start: focus on langchain core; clarify docs and experimental are separate

### PR DESCRIPTION
**Pull request by cjcjameson**
follow up to https://github.com/langchain-ai/langchain/pull/7959 , explaining better to focus just on langchain core

no dependencies

twitter @cjcjameson

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -111,7 +153,7 @@ make format_diff
 
 This is especially useful when you have made changes to a subset of the project and want to ensure your changes are properly formatted without affecting the rest of the codebase.
 
-### Linting
+#### Linting
 
 Linting for this project is done via a combination of [Black](https://black.readthedocs.io/en/stable/), [isort](https://pycqa.github.io/isort/), [flake8](https://flake8.pycqa.org/en/latest/), and [mypy](http://mypy-lang.org/).
```
Comment (leo-gan) [inline .github/CONTRIBUTING.md:158]: `Isort` and `flake8` tests run by `ruff`, as per `Makefile`

> lint lint_diff:
> 	./scripts/check_pydantic.sh .
> 	./scripts/check_imports.sh
> 	poetry run ruff .
> 	poetry run black $(PYTHON_FILES) --check
> 	poetry run mypy $(PYTHON_FILES)

So, maybe we should reference only `ruff`?

Comment (cjcjameson) [reply .github/CONTRIBUTING.md:158]: good call: https://github.com/langchain-ai/langchain/pull/1203#discussion_r1113701076

**Comment by leo-gan** [comment]
@eyurtsev Eugene, could you, please, review it?

**Review by hwchase17 — APPROVED** [review]
this is great - thanks!

**Pull request merged**